kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -29,7 +32,43 @@
         <th>{{ coreString('facilityLabel') }}</th>
       </template>
       <template #tbody>
-        <tbody>
+        <!-- On mobile, put buttons on a row of their own -->
+        <tbody v-if="windowIsSmall">
+          <template v-for="(facility, idx) in facilities">
+            <tr :key="idx" style="border: none!important">
+              <td>
+                <FacilityNameAndSyncStatus
+                  :facility="facility"
+                  :isSyncing="facilityIsSyncing(facility)"
+                  :isDeleting="facilityIsDeleting(facility)"
+                  :syncHasFailed="facility.syncHasFailed"
+                />
+              </td>
+            </tr>
+            <!-- May cause error on device with > 10000 facilities... -->
+            <tr :key="idx + 10000">
```

Comment:
I don't think this is a genuine worry, but you can always make a string key with a prefix instead `buttongroup-${idx}`